### PR TITLE
Port to OVOS

### PR DESCRIPTION
**Pull request by PureTryOut**
Mycroft has seized development due to running out of money. The OpenVoiceOS (OVOS) community has taken over development and so this commit ports the application to their stack. It's a drop-in replacement, just needs changing of imports

### CODE REVIEW

File: src/views/pair.py [hunk 1]
```diff
@@ -6,9 +6,9 @@
 from uuid import uuid4
 from requests import HTTPError
 import threading, time
-from mycroft_bus_client import Message
-from mycroft.api import is_paired
-from mycroft.identity import IdentityManager
+from ovos_bus_client import Message
+from ovos_backend_client import is_paired
+from ovos_backend_client.identity import IdentityManager
```
Comment (JarbasAl) [inline src/views/pair.py:11]: it seems this file can be deprecated, by default now ovos runs offline and pairing is not necessary

if users decide to use a personal-backend then the pairing is done manually by the user  (creating identity2.json provided by backend + enabling it in mycroft.conf)

edit: this also means ovos-backend-client dependency can be dropped

**Comment by JarbasAl** [comment]
I believe https://github.com/aspinwall-ui/lapel/blob/develop/requirements.txt also needs updating to include ovos libs

File: src/daemon.py [hunk 2]
```diff
@@ -2,10 +2,11 @@
 """
 Contains code for the message bus daemon.
 """
-from mycroft.api import DeviceApi, check_remote_pairing
-from mycroft.util.signal import create_signal
-from mycroft_bus_client import Message
-from mycroft_bus_client import MessageBusClient as OriginalMessageBusClient
+from ovos_backend_client.api import DeviceApi
+from ovos_backend_client import check_remote_pairing
+from ovos_utils.signal import create_signal
```
Comment (JarbasAl) [inline src/daemon.py:7]: note: signal utils are in the process of being deprecated in ovos, nothing is really consuming these anymore except for maybe the mk1 integration (which is broken and under rewrite anyway)

the call to create_signal can be completely removed

**Comment by knuxify** [comment]
Sorry for the late reply, I didn't really have the time to take a proper look at this until now.

As for the deprecation warnings - I can take care of those later. Just switching to the OVOS libraries is a good enough starting point.

(btw, is there a script like the old `start-mycroft` in the old Alpine Mycroft package, but for OVOS? so far I've had to start up `ovos-core`, `ovos-messagebus` etc manually, separately. I'm admittedly not even sure where the original script came from, I'd have to do some digging :p)

**Pull request merged**